bigquery/tests/unit/test__pandas_helpers.py

```diff
@@ -0,0 +1,458 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import decimal
+import functools
+
+try:
+    import pandas
+except ImportError:  # pragma: NO COVER
+    pandas = None
+try:
+    import pyarrow
+    import pyarrow.types
+except ImportError:  # pragma: NO COVER
+    pyarrow = None
+import pytest
+
+from google.cloud.bigquery import schema
+
+
+@pytest.fixture
+def module_under_test():
+    from google.cloud.bigquery import _pandas_helpers
+
```

Comment:
Nit: rm empty line?

Reply:
Thanks. We actually don't have much control over this, since `black` (Python code formatter) will just add it back.